packages/website/ts/pages/governance/governance.tsx

```diff
@@ -178,7 +138,11 @@ export class Governance extends React.Component {
                                 ))}
                             </Column>
                             <Column width="30%" maxWidth="360px">
-                                <RatingBar color="#AE5400" labels={riskLabels} rating={proposalData.risks.rating} />
+                                <RatingBar
+                                    color="#AE5400"
+                                    labels={riskLabels}
+                                    rating={this._proposalData.risks.rating}
+                                />
                             </Column>
                         </FlexWrap>
                     </SectionWrap>
```

Comment:
Can't comment on the exact line since it hasn't changed in this PR. But [152](https://github.com/0xProject/0x-monorepo/pull/1908/files#diff-180f20e32db55582c41f63e8ed73fba6R152) should map the ZEIP ID into the banner. Currently hardcoded as ZEIP-23.